pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -62,12 +62,15 @@ func (m *MultiNodeConsolidation) ComputeCommand(ctx context.Context, disruptionB
 	// would have violated the budget anyway, preserving the ordering
 	// and only considering a number of nodes that can be disrupted.
 	disruptableCandidates := make([]*Candidate, 0, len(candidates))
+	constrainedByBudgets := false
 	for _, candidate := range candidates {
 		// If there's disruptions allowed for the candidate's nodepool,
 		// add it to the list of candidates, and decrement the budget.
 		if disruptionBudgetMapping[candidate.nodePool.Name] == 0 {
 			continue
 		}
+		// set constrainedByBudgets to true if any node was a candidate but was constrained by a budget
+		constrainedByBudgets = true
```

Comment:
Does this make sense? Won't this always get called regardless if the candidate was constrained by a budget or not?